### ISSUE TITLE
Error making a POST request in the Simple To-Do List application

### ISSUE THREAD

**Issue by robertoaceves** [FoalTS/foal#274]
**Describe the bug**
The *POST* requests fail with an _Error 403: Forbidden error_ (Bad csrf token.) when trying to add a new entry in the _Simple To-Do List_.

**To Reproduce**
1. Download the source code of the first tutorial from https://foalts.org/simple-todo-list-source-code.zip.
2. Expand the file, open a Terminal and move to the _simple-todo-list_ directory
3. Install dependencies running `npm install`
4. Start the app by running `npm run develop`
5. Open _http://localhost:3000/_ in the web browser
6. Type something in the text field and press ENTER

**Expected behavior**
A new todo is added.

**Actual behavior**
The server replies with an `Error 403: Forbidden`.

**Screenshots**
Response body
![response body](https://user-images.githubusercontent.com/1446826/49127890-32bc8800-f297-11e8-9eec-3445dcabbeaa.png)

Request headers
![response headers](https://user-images.githubusercontent.com/1446826/49127927-4536c180-f297-11e8-8701-18af785c095a.png)

**Desktop:**
 - OS: MacOS 10.13.6 
 - Browser Chrome 70

**Additional context**
NA

**Comment by LoicPoullain** [comment]
Hi @robertoaceves ,

Thanks for reporting this issue!

I tried to reproduce the error downloading the zip file and it's working fine on my host. The CSRF token is correctly forwarded to the client since the `X-CSRF-Token` header is present and not empty. I wonder if the problem does not come from the session cookie which may conflict with another project that would have run on the same port 3000.

Can you try to delete it and refresh the page?
![issue github](https://user-images.githubusercontent.com/13604533/49166052-dd35b900-f332-11e8-8b18-52bda68ef10f.jpg)

A possible workaround is to disable the csrf protection in `config/settings.json` and replace the line `csrfToken: ctx.request.csrfToken(),` with `csrfToken: '',` in `view.controller.ts`

**Comment by robertoaceves** [comment]
Hi @LoicPoullain,

That did the trick! Now the app is working. Although, I had to clear all the cookies for this origin
![yum_yum](https://user-images.githubusercontent.com/1446826/49168256-18b98e80-f305-11e8-9601-4f7e65d3424a.png)

Before, I also tried disabling the CSRF protection as suggested here https://foalts.gitbook.io/docs/topic-guides/csrf-protection/csrf-protection#retreive-the-csrf-token, but it was not clear what to do with the `csrfToken: ctx.request.csrfToken(),` argument in the`src/app/controllers/view.controller.ts` file. I tried deleting it, but that created other problems.

It would be nice to add these workarounds to the tutorial.

Thanks,
Roberto
